kolibri/core/assets/src/api-resource.js

```diff
@@ -1057,7 +1057,7 @@ export class Resource {
       'color: red'
     );
     console.log(`Error occured for ${this.name} resource on page ${window.location.href}`);
-    if (store.state.route) {
+    if (store?.state?.route) {
```

Comment:
Nice - I hadn't even seen this had come in in ES2020!

Reply:
I didn't know either, but I saw this in @akolson's cookie code! Definitely will be using it